e2e/wdio/core/tests/menu.e2e-spec.ts

```diff
@@ -0,0 +1,120 @@
+import { MenuPo } from '../pages/menu.po';
+import { checkElementScreenshot,
+    click, 
+    mouseHoverElement, 
+    saveElementScreenshot, 
+    addIsActiveClass,
+    focusElement, 
+    getElementArrayLength,
+    pause,
+    waitForElDisplayed,
+    elementDisplayed,
+    doesItExist} from '../../driver/wdio';
+
+describe('Menu tests', function() {
+    const menuPage = new MenuPo();
+    const { menuButtonsArr, 
+        btnMobileMenu, 
+        menuItemsArr, 
+        dialogMobileMenu,
+        closeDialogMobileMenu,
+        dialogMenuAddonArr,
+        dialogMenuItemsArr,
+        dialogBtnBack } = menuPage;
+
+    beforeAll(() => {
+        menuPage.open();                              
+    }, 1);
+
+    it('should check RTL and LTR orientation', () => {
+        menuPage.checkRtlSwitch();
+    });
+
+    it('should check visual regression for all examples', () => {
+        menuPage.saveExampleBaselineScreenshot();
+        expect(menuPage.compareWithBaseline()).toBeLessThan(1);
+    }); 
+
+    it('should check cascading menu for all buttons', () => {
+        const buttonsArrLength = getElementArrayLength(menuButtonsArr);
+        for (let i = 0; i < buttonsArrLength; i++) {
+            click(menuButtonsArr, i);   
+            let menuItemsArrLength = getElementArrayLength(menuItemsArr);
+            for (let j = 0; j < menuItemsArrLength; j++) {
+                checkElementHoverState(menuItemsArr, 'Button'+'-'+ menuItemsArr + '-' + i +'.'+ j, menuItemsArr, j);
```

Comment:
please remove the visual regression for element hover, focus, active state checks. We are discontinuing these checks in the fundamental-ngx repo.